### ISSUE TITLE
lsof dist tar ball

### ISSUE THREAD

**Issue by kloczek** [lsof-org/lsof-linux#47]
Looks like ftp://lsof.itap.purdue.edu/pub/tools/unix/lsof/ is no longer available.
From where is possible to download last 4.91 version tar ball?

PS. do you have any plans to make soon new lsof release?

**Comment by masatake** [comment]
I don't have the 4.91 version tar ball.

**Comment by masatake** [comment]
I will try to make a 4.92 version tar ball instead. Which os do you use?

**Comment by masatake** [comment]
Though there is no tar ball, you can get the last code published by Vic with git:
```
git clone https://github.com/lsof-org/lsof-legacy.git -b master
```

**Comment by kloczek** [comment]
Don't get me wrong. I know how to generate tar ball out of git repo.
I'm only flagging that last lsof release tar ball is no longer available in the location mentioned on lsof web page.
If it is not possible to restore that tar in that situation it would be good asap new release :)

**Comment by masatake** [comment]
I'm working for making a tar ball with using "Release" feature of github.

**Comment by masatake** [comment]
See lsof-legacy repo. I'm trying to make a v4.92.1 now.

**Comment by kloczek** [comment]
Issue is that tar generated out of git repo will have different checksum than that one used in different distributions (Linux and other OSes) so recommending use that tat may be misleading or causing some alarms that source tree have been compromised.
Only this and nothing more :)
Do you know when ~ you may possibly release new lsof version? :)

**Comment by masatake** [comment]
> Issue is that tar generated out of git repo will have different checksum than that one used in different distributions (Linux and other OSes) 

I don't understand this well. Could you tell me more?

**Comment by kloczek** [comment]
Even if list of files and files content archived will be the same as long as order of the files archived in the tar will be different final checksum will be not the same as originally released 4.91.
Most of the distributions stores in distribution metadata checksums of the tar balls ans some of them are (re)checking that checksum on each (re)build with that checksum of the tar provided by source tree maintainer. Some of the distributions are raising alarms if something changed.

In other words just do not try to generate that old archive and only push to make new release asap :)

**Comment by masatake** [comment]
I see. Thank you very much.

**Comment by masatake** [comment]
Please, try https://github.com/lsof-org/lsof-legacy/releases/tag/4.92.1 .
The archive name and the top directory name in the archive are different from 4.92, the last version released by Vic. I cannot find a good way to make the names same as before.
Could you open an issue if you find something wrong in the new release.
